lib/plugins/filter/after_render/injector.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+function injectFilter(data) {
+  const { injector: _Injector } = this.extend;
+
+  function injector(data, pattern, flag, isBegin = true) {
+    return data.replace(pattern, str => {
+      if (data.includes(`hexo injector ${flag}`)) return str;
+
+      const arr = _Injector.get(flag);
+
+      if (arr.length) {
```

Comment:
If no `Injector` is registered, the `arr.length` will be `0` and nothing will be injected.

Reply:
Execution `if (arr.length)` before `data.replace`, better